src/bitvec.rs

```diff
@@ -81,6 +81,14 @@ impl Div<BitVector> for BitVector {
     }
 }
 
+impl Rem<BitVector> for BitVector {
+    type Output = BitVector;
+
+    fn rem(self, other: BitVector) -> Self::Output {
+        Self(self.0 % other.0)
```

Comment:
That does not work. You have to take care of the "n % 0" case. Essentially the same thing as in div